packages/types/lib/auth/http.api.ts

```diff
@@ -174,7 +174,7 @@ export type PostPublicSignatureAuthorization = Endpoint<{
     Params: {
         providerConfigKey: string;
     };
-    Path: '/auth/signature-based';
+    Path: '/auth/signature-based/:providerConfigKey';
```

Comment:
can we have a linter or something to test the api definition against the routes?

Reply:
ideally every endpoints have at least one integration test that would take care of that (that's why I changed it now because suddenly it was screaming at me)